### ISSUE TITLE
Proxying traefik web gui  

### ISSUE THREAD

**Issue by elebeaup** [containous/traefik#591]
Hi,

I'd like proxy HTML frontend of Træfik using Traefik. Traefik is in a container docker.
I added following labels :
- "traefik.backend=traefik"
- "traefik.port=8080"
- "traefik.frontend.rule=PathPrefixStrip:/traefik"

But, it didn't working because of redirection.

http://localhost/traefik/dashboard/ worked (The frontend is displayed) but not with the url http://localhost/traefik. The url  is forwarded to http://localhost/dashboard (instead of http://localhost/traefik/dashboard).

``` shell
$ curl http://localhost/traefik/
<a href="/dashboard/">Found</a>.
```

``` shell
$ curl http://localhost/traefik/dashboard/
<!doctype html><html ng-app="traefik"><head><meta charset="utf-8"><title>Træfɪk</title><meta name="description" content=""><meta name="viewport" content="width=device-width"><link rel="icon" type="image/png" href="traefik.icon.png"><link rel="stylesheet" href="styles/vendor-a9843e4ebf.css"><link rel="stylesheet" href="styles/app-8448ca8e46.css"></head><body><!--[if lt IE 10]>
 <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
 <![endif]--><div class="container"><header><nav class="navbar navbar-default"><div class="container-fluid"><div class="navbar-header"><a class="navbar-brand traefik-text" ui-sref="provider"><img height="16" src="traefik.icon.png"></a></div><div class="collapse navbar-collapse"><ul class="nav navbar-nav"><li><a ui-sref="provider" class="active">Providers</a></li><li><a ui-sref="health">Health</a></li></ul><ul class="nav navbar-nav navbar-right"><li><a href="https://docs.traefik.io" target="_blank">Documentation</a></li><li><a href="https://traefik.io" target="_blank"><span class="traefik-blue">traefik.io</span></a></li></ul></div></div></nav></header><main class="main"><div data-ui-view=""></div></main></div><script src="scripts/vendor-759dd4a3c8.js"></script><script src="scripts/app-ab09ab01e5.js"></script></body></html>
```

``` shell
$ curl --location http://localhost/traefik/
404 page not found
```

`

**Comment by dmytro** [comment]
I had the same problem, but was able to proxy traefik with host rule, such as `HostRegexp:traefik.{.+}.example.com`. This will need a DNS CNAME however.

**Comment by ldez** [comment]
Closed by #1638, in the version 1.3.0-rc3.
